GDNative no longer working

All I did was pulled latest commits for `Godot`

Godot Engine v3.2.alpha.custom_build.4569f5ec8 - https://godotengine.org
OpenGL ES 3.0 Renderer: GeForce GTX 1650/PCIe/SSE2
 
```
handle_crash: Program crashed with signal 11
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] /lib/x86_64-linux-gnu/libc.so.6(+0x46470) [0x7f14e5b2e470] (??:0)
[2] godot() [0xb945a8] (/home/rameshravone/Works/godot/./core/string_name.h:110)
[3] godot::ARVRAnchor::___init_method_bindings() (??:0)
[4] godot::___init_method_bindings() (??:0)
[5] godot() [0x94f836] (/home/rameshravone/Works/godot/modules/gdnative/gdnative.cpp:373)
[6] godot() [0x95da13] (/home/rameshravone/Works/godot/modules/gdnative/nativescript/nativescript.cpp:?)
[7] godot() [0x95bf8e] (/home/rameshravone/Works/godot/modules/gdnative/nativescript/nativescript.h:297)
[8] godot() [0x96afa5] (/home/rameshravone/Works/godot/./core/reference.h:277)
[9] godot() [0x2b729b3] (/home/rameshravone/Works/godot/./core/variant.h:418)
[10] godot() [0x2bfeb14] (/home/rameshravone/Works/godot/core/object.cpp:422)
[11] godot() [0x2508a38] (/home/rameshravone/Works/godot/scene/resources/resource_format_text.cpp:603)
[12] godot() [0x2e78a3e] (/home/rameshravone/Works/godot/core/io/resource_loader.cpp:199)
[13] godot() [0x967c97] (/home/rameshravone/Works/godot/modules/gdnative/nativescript/nativescript.cpp:1741)
[14] godot() [0x2e7ab01] (/home/rameshravone/Works/godot/./core/cowdata.h:370)
[15] godot() [0x2e7b717] (/home/rameshravone/Works/godot/./core/reference.h:270)
[16] godot() [0x480225] (/home/rameshravone/Works/godot/./core/cowdata.h:370)
[17] godot(main+0x8d) [0x447c1d] (/home/rameshravone/Works/godot/platform/x11/godot_x11.cpp:55)
[18] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7f14e5b0f1e3] (??:0)
[19] godot() [0x447ace] (??:?)
-- END OF BACKTRACE --
Aborted (core dumped)
make: *** [Makefile:11: rungame] Error 134
```

--- a/modules/gdnative/gdnative_api.json
+++ b/modules/gdnative/gdnative_api.json
@@ -91,6 +91,55 @@
               ["const godot_int", "p_step"],
               ["const godot_bool", "p_deep"]
             ]
+          },
+          {
+            "name": "godot_pool_byte_array_empty",
+            "return_type": "godot_bool",
+            "arguments": [
+              ["const godot_pool_byte_array *", "p_self"]
+            ]
+          },
+          {
+            "name": "godot_pool_int_array_empty",
+            "return_type": "godot_bool",
+            "arguments": [
+              ["const godot_pool_int_array *", "p_self"]
+            ]
+          },
+          {
+            "name": "godot_pool_real_array_empty",
+            "return_type": "godot_bool",
+            "arguments": [
+              ["const godot_pool_real_array *", "p_self"]
+            ]
+          },
+          {
+            "name": "godot_pool_string_array_empty",
+            "return_type": "godot_bool",
+            "arguments": [
+              ["const godot_pool_string_array *", "p_self"]
+            ]
+          },
+          {
+            "name": "godot_pool_vector2_array_empty",
+            "return_type": "godot_bool",
+            "arguments": [
+              ["const godot_pool_vector2_array *", "p_self"]
+            ]
+          },
+          {
+            "name": "godot_pool_vector3_array_empty",
+            "return_type": "godot_bool",
+            "arguments": [
+              ["const godot_pool_vector3_array *", "p_self"]
+            ]
+          },
+          {
+            "name": "godot_pool_color_array_empty",
+            "return_type": "godot_bool",
+            "arguments": [
+              ["const godot_pool_color_array *", "p_self"]
+            ]
           }
         ]
       },
@@ -1717,13 +1766,6 @@
         ]
       },
       {
-        "name": "godot_pool_byte_array_empty",
-        "return_type": "godot_bool",
-        "arguments": [
-          ["const godot_pool_byte_array *", "p_self"]
-        ]
-      },
-      {
         "name": "godot_pool_byte_array_destroy",
         "return_type": "void",
         "arguments": [
@@ -1843,13 +1885,6 @@
       {
         "name": "godot_pool_int_array_size",
         "return_type": "godot_int",
-        "arguments": [
-          ["const godot_pool_int_array *", "p_self"]
-        ]
-      },
-      {
-        "name": "godot_pool_int_array_empty",
-        "return_type": "godot_bool",
         "arguments": [
           ["const godot_pool_int_array *", "p_self"]
         ]
@@ -1979,13 +2014,6 @@
         ]
       },
       {
-        "name": "godot_pool_real_array_empty",
-        "return_type": "godot_bool",
-        "arguments": [
-          ["const godot_pool_real_array *", "p_self"]
-        ]
-      },
-      {
         "name": "godot_pool_real_array_destroy",
         "return_type": "void",
         "arguments": [
@@ -2105,13 +2133,6 @@
       {
         "name": "godot_pool_string_array_size",
         "return_type": "godot_int",
-        "arguments": [
-          ["const godot_pool_string_array *", "p_self"]
-        ]
-      },
-      {
-        "name": "godot_pool_string_array_empty",
-        "return_type": "godot_bool",
         "arguments": [
           ["const godot_pool_string_array *", "p_self"]
         ]
@@ -2241,13 +2262,6 @@
         ]
       },
       {
-        "name": "godot_pool_vector2_array_empty",
-        "return_type": "godot_bool",
-        "arguments": [
-          ["const godot_pool_vector2_array *", "p_self"]
-        ]
-      },
-      {
         "name": "godot_pool_vector2_array_destroy",
         "return_type": "void",
         "arguments": [
@@ -2372,13 +2386,6 @@
         ]
       },
       {
-        "name": "godot_pool_vector3_array_empty",
-        "return_type": "godot_bool",
-        "arguments": [
-          ["const godot_pool_vector3_array *", "p_self"]
-        ]
-      },
-      {
         "name": "godot_pool_vector3_array_destroy",
         "return_type": "void",
         "arguments": [
@@ -2498,13 +2505,6 @@
       {
         "name": "godot_pool_color_array_size",
         "return_type": "godot_int",
-        "arguments": [
-          ["const godot_pool_color_array *", "p_self"]
-        ]
-      },
-      {
-        "name": "godot_pool_color_array_empty",
-        "return_type": "godot_bool",
         "arguments": [
           ["const godot_pool_color_array *", "p_self"]
         ]

